editor/components/button/index.js

```diff
@@ -16,6 +16,7 @@ function Button( { isPrimary, isLarge, isToggled, className, ...additionalProps
 		<button
 			type="button"
 			{ ...additionalProps }
```

Comment:
Okay, that is what I thought you meant but wasn't 100% sure.  Thank you for clarifying.